src/controller/BackPostController.php

```diff
@@ -129,7 +130,7 @@ public function editPost(string $idPost, PostModel $postModel)
 		];
 
 		// display the post with error and datas form
-		$this->render('insert_post.twig', $errors, $form, $_SESSION);
+		$this->render('insert_post.twig', $errors, $form, $_SESSION, []);
```

Comment:
idem pour $post et $session

Reply:
Modif OK : 
$this->render('insert_post.twig', $_SESSION, $errors, $form);